CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(mcp23017)
+
+set (mcp23017_VERSION_MAJOR 1)
+set (mcp23017_VERSION_MINOR 0)
+
+add_library(${PROJECT_NAME} STATIC demo.c mcp23017.c mcp23017.h)
```

Comment:
we don't need demo.c in the library, drop it.

Reply:
ok